### PR TITLE
Update instructions about our logo

### PR DESCRIPTION
**Pull request by storopoli**
I made some suggestions that would address the issue raised at [discourse](https://discourse.mc-stan.org/t/trademark-protection-for-stan-name-and-logo/28629/)

### CODE REVIEW

**Comment by yizhang-yiz** [comment]
Do we want to formalize trademark guidelines before changing the instructions?

Samples of trademark policies:
https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/
https://numfocus.org/trademark-guidelines

**Comment by storopoli** [comment]
Cc @vianeylb and @spinkney.

**Comment by storopoli** [comment]
I am very positive to rust policies. That would be a good accessible language to developers and users.
We have a meeting with Arliss (NumFOCUS) on Tuesday 13th and I am going to ask her for caveats for GPL/BSD licensed software. If there's none. I am going to update this PR with Rust's template adapted to Stan so we can discuss further.

Also, if anyone wants to chime in, please do and send new commits.

**Comment by storopoli** [comment]
Sent this version as a draft to NumFOCUS to get their opinions on any legal flaws.

File: about/logo/index.md [hunk 1]
```diff
@@ -9,17 +9,18 @@ image:
...
+The Stan name and logo are registered trademarks of [NumFOCUS](http://numfocus.org/)
+under the direction of the [Stan Governing Body](/about#stan-governing-body-sgb).
 
-Both may be freely used when referencing Stan, such as in a blog post or lecture
-notes, but they may not be used in the branding of commercial entities, such
-as software packages or paid courses, without the express license of the Stan
-leadership board.  If you have any questions or would like to inquire about
-licensing the Stan name and/or logo for a commercial venture then please contact
-NumFOCUS.
+Both may be freely used when referencing Stan, such as in blog posts, lecture notes,
```
Comment (bob-carpenter) [inline about/logo/index.md:15]: Replace "Both" with "The Stan name and Stan logo" (repeating "Stan").  

"such as" -> "for example, "

File: about/logo/index.md [hunk 2]
```diff
@@ -9,17 +9,18 @@ image:
...
+or opensourced software packages, but they may not be used in the branding of commercial entities,
```
Comment (bob-carpenter) [inline about/logo/index.md:16]: "opensourced" -> "open source"

File: about/logo/index.md [hunk 3]
```diff
@@ -9,17 +9,18 @@ image:
...
+such as paid software packages or paid courses, without the express license of the Stan leadership board.
```
Comment (bob-carpenter) [inline about/logo/index.md:17]: "leadership board" -> "Governing Body"

File: about/logo/index.md [hunk 4]
```diff
@@ -9,17 +9,18 @@ image:
...
+If you have any questions or would like to inquire about licensing the Stan name
+and/or logo for a commercial venture then please contact NumFOCUS.
```
Comment (bob-carpenter) [inline about/logo/index.md:19]: This needs to say how to contact NumFOCUS.

File: about/logo/index.md [hunk 5]
```diff
@@ -9,17 +9,18 @@ image:
...
 
-When using the Stan logo please use the official versions linked below,
+Under in no circumstances our name and logo cannot be used in ways that appear (to a casual observer) official, affiliated, or endorsed by Stan, unless you have written permission from the Stan Governing Body.
```
Comment (bob-carpenter) [inline about/logo/index.md:21]: This needs to be written---it's not grammatical English and uses a confusing double negation.  The "no circumstances" is redundant.  Also, it's "may" when talking about permission vs. "can" when talking about ability.  Stan is software, so I think you mean the Stan project here.

So I'd recommend changing this to:

```
The Stan name and Stan logo may not be used in ways that suggest the usage is endorsed by the Stan project without written permission from the Stan Governing Body.
```

**Review by bob-carpenter — CHANGES_REQUESTED** [review]
This needs to be rewritten for grammar after review by NumFOCUS legal.

File: about/logo/index.md (hunk 1 above)
Comment (storopoli) [reply about/logo/index.md:15]:
```suggestion
Both may be freely used when referencing Stan, for example, in blog posts, lecture notes,
```

File: about/logo/index.md (hunk 2 above)
Comment (storopoli) [reply about/logo/index.md:16]:
```suggestion
or open source software packages, but they may not be used in the branding of commercial entities,
```

File: about/logo/index.md (hunk 3 above)
Comment (storopoli) [reply about/logo/index.md:17]:
```suggestion
such as paid software packages or paid courses, without the express license of the Stan Governing Body.
```

File: about/logo/index.md (hunk 4 above)
Comment (storopoli) [reply about/logo/index.md:19]:
```suggestion
and/or logo for a commercial venture then please contact NumFOCUS at <mailto:admin@numfocus.org>.
```

File: about/logo/index.md (hunk 5 above)
Comment (storopoli) [reply about/logo/index.md:21]:
```suggestion
The Stan name and Stan logo may not be used in ways that suggest the usage is endorsed by the Stan project without written permission from the Stan Governing Body.
```

**Comment by bob-carpenter** [comment]
This is ready to merge if it's OK with @storopoli

**Comment by storopoli** [comment]
Thanks!

**Pull request merged**